packages/dashboard/src/components/resizablePanes/index.tsx

```diff
@@ -56,9 +54,8 @@ export const ResizablePanes: FC<ResizablePanesProps> = ({ leftPane, centerPane,
   /** Current widths of the three panes, in px */
   const [leftPaneWidth, setLeftPaneWidth] = useState(DEFAULT_SIDE_PANE_WIDTH);
   const [rightPaneWidth, setRightPaneWidth] = useState(DEFAULT_SIDE_PANE_WIDTH);
-  const selectedWidget = useSelector((state: DashboardState) => state.selectedWidgets);
-  const [isRightPaneCollapsed, setRightPaneCollapsed] = useState(selectedWidget?.length === 0);
-  const [isLeftPaneCollapsed, setLeftPaneCollapsed] = useState(selectedWidget?.length === 0);
+  const [isRightPaneCollapsed, setRightPaneCollapsed] = useState(true);
```

Comment:
what about using `import useLocalStorage from 'react-use/lib/useLocalStorage';` instead? We can preserve the state this way?

Reply:
as per the new requirement we are not persisting the panel open/closed status. question is raised in the team, based on the product call, i'll update this.

Reply:
updated with default open for resource explorer and close for configuration panel. currently not preserving the panel status, so useLocalStorage not used.